standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClientWithLocalCache.java

```diff
@@ -214,4 +214,13 @@ protected PartitionsSpecByExprResult getPartitionsSpecByExprResult(PartitionsByE
 
     return r;
   }
+
+  private boolean isRequestCachable(Object request) {
```

Comment:
Add a comment explaining what sort of request we are trying to cache and avoid caching.